test/CMakeLists.txt

```diff
@@ -43,6 +43,18 @@ if (NOT EXCLUDE_FP)
     list(APPEND DC_SRC ${FP_SRC})
 endif ()
 
+
+if (NOT EXCLUDE_SHADOW)
+    if (NOT EXCLUDE_CONFIG_SHADOW)
+        file(GLOB CONFIG_SHADOW_SRC "../source/shadow/ConfigShadow.cpp")
+        list(APPEND DC_SRC ${CONFIG_SHADOW_SRC})
+    endif ()
+    if (NOT EXCLUDE_CONFIG_SHADOW)
```

Comment:
Wrong option tag. It should be `EXCLUDE_SAMPLE_SHADOW`.

Reply:
updated